excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SearchCriteriaValidatorTest.java

```diff
@@ -0,0 +1,162 @@
+
+    private static final String TEST_EXPRESSION_OTHER_CASE_REFERENCE_1 = "OtherCaseReference1";
+    private static final String TEST_EXPRESSION_OTHER_CASE_REFERENCE_2 = "OtherCaseReference2";
+
+    private static final String TEST_EXPRESSION_BAD = "BadTestExpression";
+
+    @Mock
+    private DotNotationValidator dotNotationValidator;
+
+    @Mock
+    private ParseContext parseContext;
+
+    @InjectMocks
+    private SearchCriteriaValidator searchCriteriaValidator;
+
+    @DisplayName("should validate OtherCaseReference field")
+    @Test
+    void shouldValidateOtherCaseReferenceField() {
+
+        // GIVEN
+        SearchCriteriaEntity searchCriteriaEntity1 = createBlankSearchCriteriaEntity();
+        searchCriteriaEntity1.setOtherCaseReference(TEST_EXPRESSION_OTHER_CASE_REFERENCE_1);
+
+        // WHEN
+        searchCriteriaValidator.validate(List.of(searchCriteriaEntity1), parseContext);
+
+        // THEN
+        verifyDotNotationValidatorCallMadeFor(ColumnName.OTHER_CASE_REFERENCE, TEST_EXPRESSION_OTHER_CASE_REFERENCE_1);
+    }
+
+    @DisplayName("should validate multiple SearchCriteriaEntity values")
+    @Test
+    void shouldValidateMultipleSearchCriteriaEntities() {
+
+        // GIVEN
+        SearchCriteriaEntity searchCriteriaEntity1 = createBlankSearchCriteriaEntity();
+        searchCriteriaEntity1.setOtherCaseReference(TEST_EXPRESSION_OTHER_CASE_REFERENCE_1);
+
+        SearchCriteriaEntity searchCriteriaEntity2 = createBlankSearchCriteriaEntity();
+        searchCriteriaEntity2.setOtherCaseReference(TEST_EXPRESSION_OTHER_CASE_REFERENCE_2);
+
+        // WHEN
+        searchCriteriaValidator.validate(List.of(searchCriteriaEntity1, searchCriteriaEntity2), parseContext);
+
+        // THEN
+        verifyDotNotationValidatorCallMadeFor(ColumnName.OTHER_CASE_REFERENCE, TEST_EXPRESSION_OTHER_CASE_REFERENCE_1);
+        verifyDotNotationValidatorCallMadeFor(ColumnName.OTHER_CASE_REFERENCE, TEST_EXPRESSION_OTHER_CASE_REFERENCE_2);
+    }
+
+    @DisplayName("should validate as ok even if OtherCaseReference is blank")
+    @Test
+    void shouldValidateAsOkEvenIfOtherCaseReferenceIsBlank() {
+
+        // GIVEN
+        SearchCriteriaEntity searchCriteriaEntity1 = createBlankSearchCriteriaEntity();
+        searchCriteriaEntity1.setOtherCaseReference("");
+
+        SearchCriteriaEntity searchCriteriaEntity2 = createBlankSearchCriteriaEntity();
+        searchCriteriaEntity2.setOtherCaseReference(null);
+
+        // WHEN
+        searchCriteriaValidator.validate(List.of(searchCriteriaEntity1, searchCriteriaEntity2), parseContext);
+
+        // THEN
+        verifyDotNotationValidatorCallNeverMadeFor(ColumnName.OTHER_CASE_REFERENCE);
+    }
+
+    @DisplayName("throws exception if OtherCaseReference validation fails")
+    @Test
+    void throwsExceptionIfOtherCaseReferenceValidationFails() {
+
+        // GIVEN
+        SearchCriteriaEntity searchCriteriaEntity1 = createBlankSearchCriteriaEntity();
```

Comment:
remove `1` from name ?

Reply:
done in [59c375d1](https://github.com/hmcts/ccd-definition-store-api/pull/1061/commits/59c375d172d81b375f2fe43039955b022afd906e)